package/MDAnalysis/coordinates/memory.py

```diff
@@ -502,8 +502,12 @@ def timeseries(self, asel=None, start=0, stop=-1, step=1, order='afc'):
             data is returned whenever `asel` is different from ``None``.
         start : int (optional)
         stop : int (optional)
+            Note that `stop` is currently *inclusive* but will be deprecated in
+            favour of being *exclusive* in 3.0.  
         step : int (optional)
-            range of trajectory to access, `start` and `stop` are *inclusive*
+            range of trajectory to access, `start` and `stop` are
```

Comment:
Just remove the inclusive Blabla from here, it never made sense under step anyway. The only place that changes is stop.